test/L1TokenBridge.t.sol

```diff
@@ -193,4 +201,78 @@ contract L1TokenBridgeTest is DssTest {
         assertEq(l1Token.balanceOf(escrow), 0);
         assertEq(l1Token.balanceOf(address(0xced)), 100 ether);
     }
+
```

Comment:
We should add the authed inherited methods to `testAuthModifiers`

Reply:
Hmmm ok we have separate functions for this. Probably not that simple to add in the array.

Reply:
It actually works fine adding `bridge.upgradeToAndCall.selector` to the test, we could then remove this one.

Reply:
Done. Let's remember to do the same for the other relevant repos at some point.